test/python/conftest.py

```diff
@@ -22,12 +21,7 @@
 
 from tensorflow.python.framework import ops
 
-from common import LIBNGRAPH_DEVICE
-
-
-@pytest.fixture(scope='session', autouse=True)
-def load_ngraph_device():
-  cdll.LoadLibrary(os.path.join('../../src', LIBNGRAPH_DEVICE))
+import ngraph  # noqa
```

Comment:
We cannot do this at the unit test level. The unit tests must not require the Python package (which is built using `make install` that logically comes after running the tests and ensuring they all pass. Please revert his change.

Reply:
Ack.  Okay.